README.md

```diff
@@ -7,16 +7,54 @@ See [wasix.org](https://wasix.org) for more.
 
 ## Installation
 
-To install this Cargo subcommand, first you'll want to [install
-Rust](https://www.rust-lang.org/tools/install) and then you'll execute:
+You can install this Cargo subcommand via:
 
-```
+> Note: Installation requires [Rust](<(https://www.rust-lang.org/tools/install)>) be installed on your system.
```

Comment:
Should probably also mention that rustup is required?

Reply:
rustup is now mentioned